3rdParty/s2geometry/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build shared libraries" FORCE)
 set(BUILD_EXAMPLES OFF CACHE BOOL "Build examples" FORCE)
 set(INSTALL_HEADERS OFF CACHE BOOL "Install headers" FORCE)
 
-add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/${ARANGO_S2GEOMETRY_VERSION} EXCLUDE_FROM_ALL)
+add_subdirectory(${ARANGO_S2GEOMETRY_VERSION})
```

Comment:
It's not correct without this https://github.com/arangodb/arangodb/blob/devel/CMakeLists.txt#L1190